test/controllers/organizations/staff/adoption_application_reviews_controller_test.rb

```diff
@@ -132,4 +143,27 @@ class Organizations::Staff::AdoptionApplicationReviewsControllerTest < ActionDis
       end
     end
   end
+
+  context "Viewing application details" do
+    setup do
+      @user = create(:admin)
+      sign_in @user
+
+      @person = create(:person)
```

Comment:
Here I think we can actually focus on the `User` resource. So, you can create an `adopter` (see the factory for `User` if not sure what I mean). Then, I believe that when we create a `User` record it will automatically create a `Person` (because user belongs to person). Then, you can use that `Person` record instead of creating a new one. I believe you will need to keep the rest here to create the `FormSubmission` etc.

Reply:
Yes, I checked the User factory and found that adopter already creates form submission so updated code accordingly.